content/html/concepts/elements/elements.md

```diff
@@ -77,3 +77,5 @@ The following example contains four HTML elements: `<html>`, `<body>`, `<h1>`, a
   </body>
 </html>
 ```
+
```

Comment:
I'd like a sentence introducing the image.

Reply:
No problem at all, I'll add that in.

Reply:
@SSwiniarski does the sentence that I have added resolve your change request?